tarpc/src/macros.rs

```diff
@@ -261,7 +261,7 @@ macro_rules! service {
 #[doc(hidden)]
 #[macro_export]
 macro_rules! service_inner {
-    // Pattern for when the next rpc has an implicit unit return type
+// Pattern for when the next rpc has an implicit unit return type
```

Comment:
I know they're definitely worse but it's cargo fmt and I wanted to start using the hook lol.